### PR TITLE
Pull back deleted files

### PR DESCRIPTION
**Pull request by mamkhari**
This reverts commit d5886053b79158540c4b4d68b8e0ec2b86f07d8c.

### CODE REVIEW

**Comment by mmphego** [comment]
Please address the merge conflicts.
Watch this helpful video for understanding: https://youtu.be/JtIX3HJKwfo

**Comment by mamkhari** [comment]
I addressed some merge conflicts. May you please review.

File: ToDo.md [hunk 1]
```diff
@@ -2,59 +2,81 @@
...
-### 16 March 
-
-- [x] Coursera online course (Python for everybody)
-- [x] Create a new repository to track  work progress.
-- [x] Create a new branch from existing repository (extract_sensor_names) and refactor your project (extract_unique_sensors.py).
-- [x] Create functions that calculate force of gravity, potential energy and equation of a straight line(This is to get basic understanding on how to use functions).Then Convert my project to "functions"
+- [ ] Coursera online course (Python for everybody)
+- [ ] Create a new repository to track  work progress.
+- [ ] Create a new branch from existing repository (extract_sensor_names) and refactor your project (extract_unique_sensors.py).
+- [] Create functions that calculate force of gravity, potential energy and equation of a straight line(This is to get basic understanding on how to use functions).Then Convert my project to "functions"
```
Comment (mmphego) [inline ToDo.md:14]: Please fix this, there should be a space between the braces

File: ToDo.md [hunk 2]
```diff
@@ -2,59 +2,81 @@
...
+- Read up about CI/CD
+- Devops (tools)
+- Docker/Jenkins
+
+### 31 March
 
-### 31 March 
 - [ ] Address comments from a reviewed pull request
-- [x] Sign up for Udemy course "Automate the Boring Stuff with Python Programming."
+- [x] Sign up for Udemy course "Automate the Boring Stuff   with Python Programming."
```
Comment (mmphego) [inline ToDo.md:36]: Link please: See https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

Bookmark that link or star the repository

File: ToDo.md [hunk 3]
```diff
@@ -2,59 +2,81 @@
...
 - [x] Attend classes for Automate the Boring Stuff with Python Programming.
 
-### 15 April 
+### 15 April
+
 - [ ] Enroll for "Python Data Structures"
-- [x]I will cover chapter 6 and chapter 7 of the course. ( Strings and Files)
-### 16 April 
-- Cover chapter 8 of "Python Data Structures."
+- [x] I will cover chapter 6 and chapter 7 of the course. (Strings and Files)
```
Comment (mmphego) [inline ToDo.md:49]: We got no idea what chapter 6 and 7 is, Add links

File: ToDo.md [hunk 4]
```diff
@@ -2,59 +2,81 @@
...
+### 16 April
+
+- [ ] Cover chapter 8 of "Python Data Structures."
+
 ### 20 April
-- [x] Address comments   
-- [x] Do chapter 9 of "Python Data Structures" 
-### 21 April 
+
+- [x] Address comments
```
Comment (mmphego) [inline ToDo.md:57]: What comments - you need to be more specific. Add PR or link

File: ToDo.md [hunk 5]
```diff
@@ -2,59 +2,81 @@
...
+- [x] Do chapter 9 of "Python Data Structures"
```
Comment (mmphego) [inline ToDo.md:58]: Links!!!

File: ToDo.md [hunk 6]
```diff
@@ -2,59 +2,81 @@
...
+
 - [ ] Enroll for "Python Data Structures"
-- [x]I will cover chapter 6 and chapter 7 of the course. ( Strings and Files)
-### 16 April 
-- Cover chapter 8 of "Python Data Structures."
+- [x] I will cover chapter 6 and chapter 7 of the course. (Strings and Files)
+
+### 16 April
+
+- [ ] Cover chapter 8 of "Python Data Structures."
```
Comment (mmphego) [inline ToDo.md:53]: Links!!!

File: learning_progress/Done.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
+# What I did
```
Comment (mmphego) [inline learning_progress/Done.md:1]:
```suggestions
# What I have done.
```

File: learning_progress/Done.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+## February 2020
+
+### 28 February
+
+1. Created a new git repository to track my work progress.(<https://github.com/mamkhari/Progress>)
+2. Refactored my piece of code and addressed the comments on the PR request.(<https://github.com/mamkhari/extract_sensor_names/pull/3>)
+3. Revised chapter 4: Functions of PY4E.(<https://www.py4e.com/html3/04-functions>)
```
Comment (mmphego) [inline learning_progress/Done.md:9]: See the markdown cheat sheet link wrt linking

**Comment by mmphego** [comment]
Please consider doing this 3hrs course on markdown and readmes: https://www.udacity.com/course/writing-readmes--ud777

**Pull request merged**